tracer/test/Datadog.Trace.Tests/Configuration/Telemetry/ConfigurationBuilderTests.cs

```diff
@@ -176,7 +176,7 @@ public void RecordsTelemetryCorrectly(object value)
                     },
                     { } i => new List<Entry>
                     {
-                        Entry.String(Key, i.ToString(), ConfigurationOrigins.Code, error: null),
+                        Entry.String(Key, Convert.ToString(i, CultureInfo.InvariantCulture), ConfigurationOrigins.Code, error: null),
```

Comment:
nit: I don't think you need to use `Convert` do you? 🤔 
```suggestion
                        Entry.String(Key, i.ToString(CultureInfo.InvariantCulture), ConfigurationOrigins.Code, error: null),
```

Reply:
Your option was my first choice, but that overload is missing in some frameworks.

Reply:
[It's there since .NET Framework 1.0 AFAICT](https://learn.microsoft.com/en-us/dotnet/api/system.int32.tostring?view=net-8.0#system-int32-tostring(system-iformatprovider)) 😕